packages/gatsby-theme-newrelic/README.md

```diff
@@ -2653,74 +2652,38 @@ With formatting options:
 const formattedCode = useFormattedCode(code, { printWidth: 100 });
 ```
 
-### `useInstrumentedData`
-
-A hook that instruments raw data with New Relic Browser.
-
-```js
-import { useInstrumentedData } from '@newrelic/gatsby-theme-newrelic';
-```
-
-**Arguments**
-
-- `attributes` _(object)_: Data passed to the
-  [`newrelic.addPageAction`](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-agent-spa-api/add-page-action)
-  API when called. These attributes **MUST** contain an `actionName` property,
-  otherwise the data will not be instrumented. All other attributes will be
-  attached to the `attributes` property of the page action.
-- `options` _(object)_: Options for the hook
-  - `enabled` _(boolean)_: Determines whether the data should be instrumented
-    via `newrelic.addPageAction`. Set to `false` to disable instrumentation.
-    **DEFAULT**: `true`
-
-**Returns**
-
-`Void`
-
-**Examples**
-
-```js
-const MyComponent = ({ searchTerm, onChange }) => {
-  useInstrumentedData(
-    { actionName: 'search', searchTerm },
-    { enabled: Boolean(searchTerm) }
-  );
-
-  return <input value={searchTerm} onChange={onChange} />;
-};
-```
-
 ### `useInstrumentedHandler`
 
-A hook that wraps a function handler with New Relic Browser instrumentation.
+A hook that wraps a function handler with Tessen instrumentation.
 
 ```js
 import { useInstrumentedHandler } from '@newrelic/gatsby-theme-newrelic';
 ```
 
 **Arguments**
 
-- `handler` _(function)_: The function hander that should be augmented with New
-  Relic Browser instrumentation. This can be `null` or `undefined`.
-- `attributes` _(object | function)_: Data passed to the
-  [`newrelic.addPageAction`](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-agent-spa-api/add-page-action)
-  API when called. The attributes **MUST** contain an `actionName` property,
-  otherwise the handler will not be instrumented. All other attributes will be
-  attached to the `attributes` property of the page action. You can pass a
-  function to instrument dynamic data. If this is a function, the function will
-  be called with the same arguments passed to the handler.
+- `handler` _(function)_: The function hander that should be augmented with Tessen instrumentation. 
+  This can be `null` or `undefined`.
+- `attributes` _(object | function)_: Data passed to the `Tessen.track` API when called. 
+  The attributes **MUST** contain...
+
+  - `tessenEventName` - Needs to be in [Camel Case](https://en.wikipedia.org/wiki/Camel_case)
+  - `tessenCategoryName` - Needs to be in [Title Case](https://en.wikipedia.org/wiki/Title_case)
```

Comment:
I don't think we need to prepend these variables with `tessen` since the extra context isn't necessary (we're in a tessen track block, we can assume they're for tessen).

Furthermore, according to [the docs](https://pages.datanerd.us/commune/project-tessen/#globally-required-attributes) these variables are `eventName` and `category`. We might want to use those names to stay consistent.